src/theme/ThemeProvider.js

```diff
@@ -111,7 +111,6 @@ const hackcampTheme = {
       googleHover: '#fafafa',
       githubHover: '#545159',
     },
-    greetingBorder: '#000000',
     applicationCard: 'rgba(75, 65, 130, 0.2)', // TODO
```

Comment:
I think there's more instances of `greetingBorder` in ThemeProvider, can we remove all of them? Can we also remove `applicationCard` since it's not being used anymore? :))

Reply:
ooo ok word